pkg/api/types.go

```diff
@@ -1058,6 +1058,8 @@ type PodConditionType string
 
 // These are valid conditions of pod.
 const (
+	// PodScheduled represents status of the scheduling process.
```

Comment:
It's a little redundant, but maybe "status of the scheduling process for this pod"